tests/test_mxacquisition.py

```diff
@@ -4,6 +4,20 @@
 
 import ispyb
 import pytest
+import json
+import io
+import gzip
+import binascii
+
+
+def gzip_json(obj):
+    json_str = json.dumps(obj)
+
+    out = io.BytesIO()
+    with gzip.GzipFile(fileobj=out, mode="w") as fo:
+        fo.write(json_str.encode())
+
+    return out.getvalue()
```

Comment:
Any particular reason why this is not just
```suggestion
import binascii
import json
import gzip


def gzip_json(obj):
    return gzip.compress(json.dumps(obj).encode())
```

Reply:
As long as that produces exactly the same binary thats fine (this is read in other places). Dont think id realised you could compress a string directly.

Reply:
we'll see if tests still pass :smile: 

(I've checked with simple data structures, so it should be fine)